### PR TITLE
Add persistent configuration details to Docker docs

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Related to https://github.com/netdata/netdata/issues/5516#issuecomment-687791386

This PR adds more details about configuring Netdata when it's run in a Docker container. The recommended method is using a bind mount, which doesn't require any additional steps. I also added details about editing the config _inside_ the container by first using `docker exec`.

I also added a new section using @ilyam8's instructions to populate a volume for a host-editable config directory.

##### Component Name

area/docs

### CODE REVIEW

**Review by prologic — APPROVED** [review]

File: packaging/docker/README.md [hunk 1]
```diff
@@ -87,8 +93,6 @@ volumes:
   netdatacache:
```
Comment (ilyam8) [inline packaging/docker/README.md:93]:
```yaml
...
volumes:
  netdatalib:
  netdatacache:
```

=>

```yaml
volumes:
  netdataconfig:
  netdatalib:
  netdatacache:
```

Comment (joelhans) [reply packaging/docker/README.md:94]: Good catch, thank you very much! Just added this.

**Review by prologic — APPROVED** [review]

**Comment by prologic** [comment]
As these are docs, they can go into `master`.

**Comment by joelhans** [comment]
Thanks, @prologic! Merging now.

**Comment by joelhans** [comment]
[Improve docker configuration docs](https://trello.com/c/SzphqF8G/283-improve-docker-configuration-docs)

**Pull request merged**